platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/PkiTools.java

```diff
@@ -243,13 +268,12 @@ public static X500Name makeDistinguishedName(String commonName) {
     public static X509Certificate pemToCertificate(String certString) {
         CertificateFactory cf = new CertificateFactory();
         ByteArrayInputStream in = new ByteArrayInputStream(PkiTools.pemToDer(certString));
-        X509Certificate cert = null;
+        X509Certificate cert;
```

Comment:
In this case, the null was unnecessary. It isn't IntelliJ complaining about the missing null, it is javac as the value of the variable is indeterminate unless initialized.